resources-generator/src/main/kotlin/dev/icerock/gradle/actions/apple/CopyResourcesFromKLibsAction.kt

```diff
@@ -71,7 +71,7 @@ internal abstract class CopyResourcesFromKLibsAction : Action<Task> {
         val layout: KotlinLibraryLayout = getKotlinLibraryLayout(klibFile)
         return layout.resourcesDir.listFilesOrEmpty
             .filter { it.isDirectory && it.extension == "bundle" }
-            .map { it.javaFile() }
+            .map { File(it.path) }
```

Comment:
The `javaFile()` extension function seems to have also been removed in Kotlin 2.2.0.